core/src/PrognosticData.cpp

```diff
@@ -15,16 +15,23 @@
 
 namespace Nextsim {
 
+static const std::string checkFieldsKey = "debug.check_fields";
+static const std::string checkFieldsFastKey = "debug.check_fields_fast";
+static const std::map<int, std::string> keyMap
+    = { { PrognosticData::CHECKFIELDS_KEY, checkFieldsKey },
+          { PrognosticData::CHECKFIELDSFAST_KEY, checkFieldsFastKey } };
+static constexpr bool checkFieldsDefault = false;
+static constexpr bool checkFieldsFastDefault = true;
```

Comment:
This should be moved to the `CheckingModelComponent` class. There would need to be a way for subclasses to provide their own configuration prefix and a call to `CheckingModelComponent::configure()` in the `configure()` function of each subclass.
`CheckingModelComponent`:
```
std::string configurationPrefix() virtual = 0;
…
configure()
{
…
    boolCheckAll() = Configured::getConfiguration( configurationPrefix() + checkAllKey, checkFieldsDefault);
    boolCheckFields
        = Configured::getConfiguration(configurationPrefix() + checkFieldsKey, checkFieldsFastDefault);
```
Also, common text for the `help-config` test could be provided by `CheckingModelComponent`, to be used by all subcalsses.